packages/design-system/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,104 @@
+import { ClearIcon } from '../ClearIcon';
+import React from 'react';
+import classNames from 'classnames';
+
+const actionableKeys = ['Enter', 'Backspace', 'Delete'];
+const idFromValue = (value: string) => `${value}-tag`;
+
```

Comment:
To stay consistent with other components, can we use `uniqueId` from lodash instead. See `Alert.jsx` for an example.